frontend/packages/core/src/WorkflowLayout/index.tsx

```diff
@@ -90,22 +112,29 @@ const WorkflowLayout = ({
       !workflowPaths.find(path => !!matchPath({ path }, url))
   );
 
+  if (variant === "custom") {
+    return <>{children}</>;
+  }
+
   return (
     <LayoutContainer $variant={variant}>
       {!hideHeader && (
         <PageHeader $variant={variant}>
-          <Breadcrumbs entries={breadcrumbsEntries} />
-          <PageHeaderMainContainer>
-            {heading && (
-              <>
-                {React.isValidElement(heading) ? (
-                  heading
-                ) : (
-                  <Heading variant="h2">{heading}</Heading>
+          <PageHeaderBreadcrumbsWrapper $onlyBreadcrumbs={onlyBreadcrumbs}>
+            <Breadcrumbs entries={breadcrumbsEntries} />
+          </PageHeaderBreadcrumbsWrapper>
+          {!onlyBreadcrumbs && (
+            <PageHeaderMainContainer>
```

Comment:
If no `title` and `subtitle` should the containers be rendered?

Reply:
Done in [84aacb4](https://github.com/lyft/clutch/pull/3156/commits/84aacb421d6b0d9ebdabae52825012747ef654e2).